js/src/widgets/contextControls.js

```diff
@@ -69,11 +175,37 @@
                                    '<a class="mirador-osd-edit-mode hud-control" role="button" aria-label="Make a new annotation using mouse">',
                                    '<i class="fa fa-lg fa-edit"></i>',
                                    '</a>',
-                                   '{{/if}}',
-                                   '<a class="mirador-osd-refresh-mode hud-control" role="button" aria-label="Refresh annotations">',
+                                   '<a class="hud-control draw-tool" style="color:#abcdef;">',
+                                   '|',
+                                   '</a>',
+                                   '{{#each tools}}',
```

Comment:
If the default `availableAnnotationDrawingTools ` is the full set of tools we can ignore inappropriate developer settings like no array or empty array (else it is a sub-set of the tools).